lib/pbench/cli/server/shell.py

```diff
@@ -194,6 +195,32 @@ def main():
         logger.info("Created database {}", db_uri)
     Database.init_db(server_config, logger)
 
+    # Add the Pbench Server binaries to the PATH.
+    os.environ["PATH"] = ":".join([str(server_config.BINDIR), os.environ["PATH"]])
+
+    crontab_f = Path(crontab_dir) / "crontab"
+    if not crontab_f.exists():
+        # Create the crontab file from the server configuration.
+        cp = subprocess.run(
+            ["pbench-create-crontab", crontab_dir],
+            cwd=server_config.log_dir,
+        )
+        if cp.returncode != 0:
+            logger.error(
+                "Failed to create crontab file from configuration: {}", cp.returncode
+            )
+            crontab_f.unlink(missing_ok=True)
+            sys.exit(1)
```

Comment:
I don't think this is relevant for this PR, but since I've already made other comments that I'll need to resolve ... given that this internal Python use is now the only invocation of this `pbench-create-crontab` command, we really ought to consider streamlining it a bit by making it a Python utility method that can simply be *called* inline instead of firing off a subprocess.

Reply:
I thought about making that kind of change, but figured it'd be best approached in another PR.

Reply:
Should we create an issue for this work?